src/cod/stats.ts

```diff
@@ -87,23 +87,30 @@ function sum(stats, field): number {
 }
 
 function calculateStats(matches): Stats {
-    let stats = matches.map(x => x.segments[0].stats);
-    let statValues: Stats = {
+    const stats = matches.map(x => x.segments[0].stats);
+    const statValues: Stats = {
         'Matches': stats.length,
         'Kills': sum(stats, 'kills'),
         'Deaths': sum(stats, 'deaths'),
         'Assists': sum(stats, 'assists'),
         'Time Played': formatDuration(sum(stats, 'timePlayed')),
         'Avg. Game Time': formatDuration(sum(stats, 'timePlayed') / stats.length),
-        'Avg. Team Placement': parseInt((sum(stats, 'teamPlacement') / Math.max(stats.length, 1)).toString()),
+        'Avg. Team Placement': Math.floor(sum(stats, 'teamPlacement') / stats.length),
```

Comment:
We should probably convert this to a percentile as the team placement number doesnt make sense when multiple modes (solos-quads) are played within the specified time frame.

So if you placed 100/150 in solos (33rd percentile) and 20/50 in trios (60th percentile), your overall percentile would be 46.5 percentile. 

I hope my percentile math isn't all wrong 😅

Reply:
PR has nothing to do with the stat itself and only fixes some bad code related to int parsing. We can revisit this in a different issue.